x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_import.md

```diff
@@ -215,25 +238,28 @@ CASE: should have the same outcome for all rule types
 **Automation**: 1 integration test.
 
 ```Gherkin
-Given the import payload contains modified and unmodified, custom and prebuilt rules
-When the user imports the rule
+Given the import payload contains prebuilt non-customized, prebuilt customized, and custom rules
+When the user imports these rules
 Then custom rules should be created or updated, with versions defaulted to 1
 And prebuilt rules should be created or updated,
 And prebuilt rules missing versions should be rejected
 ```
 
-#### Scenario: Importing prebuilt rules when the rules package is not installed
+#### Scenario: Importing a prebuilt rule when the rules package is not installed
 
 **Automation**: 1 integration test.
 
 ```Gherkin
-Given the import payload contains prebuilt rules
-And no rules package has been installed locally
+Given the import payload contains a prebuilt rule
+And its rule_id matches one or a few rule assets from the latest package
+And the package hasn't been installed yet
 When the user imports the rule
-Then all rules should be created or updated as custom rules
+Then the latest package should get installed automatically
+And the rule should be created or updated
+And the ruleSource type should be "external"
```

Comment:
In this test plan, we use a lot of technical terminology that is obscure to anyone otside of our team and can become obsolete at some point.

I'd suggest to replace it with a more user-friendly terminology:

- "rule source type is external" to "prebuilt rule"
- "rule source type is internal" to "custom rule"
- "isCustomized is true" to "marked as customized"
- etc

I will address this in a separate PR.

Reply:
PR: https://github.com/elastic/kibana/pull/213434